lib/situation_room/sites.ex

```diff
@@ -1,8 +1,27 @@
 defmodule SituationRoom.Site do
+
   use Ecto.Schema
+  import Ecto.Changeset
 
   schema "sites" do
     field :endpoint, :string
     field :name, :string
   end
+
+  # @spec changeset(
+  #         {map, map}
+  #         | %{
+  #             :__struct__ => atom | %{:__changeset__ => map, optional(any) => any},
+  #             optional(atom) => any
+  #           },
+  #         :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
+  #       ) :: Ecto.Changeset.t()
+  def changeset(struct, params) do
```

Comment:
Let's use something a little more specific than just `struct` as the parameter name here.  We know/expect it to be an instance of a Site struct, so how about just calling the parameter `site` here?